dmutils/forms/mixins.py

```diff
@@ -58,6 +58,12 @@ class DMFieldMixin:
     '''
     def __init__(self, label=None, validators=None, hint=None, question_advice=None, **kwargs):
         super().__init__(label=label, validators=validators, **kwargs)
+
+        if "id" in kwargs:
+            self.href = kwargs["id"]
+        else:
+            self.href = "input-" + self.id
+
```

Comment:
Have we tested if there is any weird recursiveness with this mixin or the selection one? I am happy to fire it up and have a go at it, if not.

Reply:
Please have a go at it :)

Reply:
just saw the comments above too!

Reply:
remind me, which page is best to test this?